BrainPortal/app/models/ssh_data_provider.rb

```diff
@@ -202,6 +210,7 @@ def impl_provider_collection_index(userfile, directory = :all, allowed_types = :
       if userfile.is_a? FileCollection
         if directory == :all
           entries = sftp.dir.glob(provider_full_path(userfile).to_s, "**/*")
+          #entries = sftp.dir.glob(provider_full_path(userfile).to_s, "**/*", File::FNM_DOTMATCH)
```

Comment:
Should it be in remove ?

Reply:
Same thing about the behavior I was saying earlier.